lib/build.js

```diff
@@ -517,6 +527,36 @@ class BuildModel extends BaseModel {
         return ['ABORTED', 'FAILURE', 'SUCCESS'].includes(this.status) ||
         (this.status === 'UNSTABLE' && !!this.endTime);
     }
+
+    /**
+     * getMetrics for this build
+     * @method getMetrics
+     * @param  {Object}   [config]              Configuration object
+     * @param  {String}   [config.startTime]    Look at steps created after this startTime
+     * @param  {String}   [config.endTime]      Look at steps created before this endTime
+     * @return {Promise}  Resolves to array of metrics for steps belong to this event
+     */
+    async getMetrics(config) {
```

Comment:
```suggestion
    async getMetrics({ startTime=null, endTime=null }) {
```

Does this work?
And you can remove lines 541, 542

Reply:
Ah doesn't let me do exactly that. It seems default param value in es6 will not work on JSON, it will give error `Unable to deconstruct endTime` if the field `endTime` is missing. 
However, found something similar:
```
async getMetrics(config = { startTime: null, endTime: null }) {
```
then need to reference it as `config.startTime` instead of `startTime`. 
Let me know if there are better ways.

Reply:
Addressed here: https://github.com/screwdriver-cd/models/pull/324/commits/3b0ffbd5ce81304543b0142127426523148e6183

Reply:
Aw that's weird that it didn't work :\
Your workaround looks good to me